.brew_install_gcc6.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
```

Comment:
Could you include comments/descriptions at the top of each file to just explain in a verbose way which versions of the compilers you are setting up?